packages/react-components/react-shared-contexts/src/AvatarContext.ts

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
```

Comment:
It's up to you, but I think that it should be hosted in a separate package as `portal-compat-context`.

Reply:
Or is there any real issue with including it in the `react-avatar` package? The rest of `react-avatar` should be tree shaken out of `react-table` if it's not used, right?